index.bs

```diff
@@ -2630,7 +2630,7 @@ The sample code for generating and registering a new key follows:
             algorithm: "RS256"
         }
     ];
-    var challenge = "Y2xpbWIgYSBtb3VudGFpbg";
+    var challenge = Uint8Array.from("climb a mountain", c=>c.charCodeAt(0));
```

Comment:
These match the old values, but it might make more sense to show random 128-bit strings for the challenges (https://w3c.github.io/webcrypto/Overview.html#Crypto-method-getRandomValues) and more likely ids for the acceptableCredentials?